package.json

```diff
@@ -1,19 +1,38 @@
 {
-  "name": "camera-widget-for-phonegap",
-  "version": "3.2.0",
+  "name": "CameraWidgetForPhoneGap",
+  "version": "3.3.0",
   "description": "Enable native camera functionality within your Mendix hybrid app",
+  "license": "Apache-2.0",
+  "author": "Mendix BV",
+  "private": true,
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mendix/CameraWidgetForPhoneGap.git"
-  },
-  "author": "Mendix Technology bv",
-  "license": "Apache-2.0",
-  "bugs": {
-    "url": "https://github.com/mendix/CameraWidgetForPhoneGap/issues"
+    "url": "https://github.com/mendix/CameraWidgetForPhoneGap"
```

Comment:
incorrect should be `"url": "https://github.com/mendix/CameraWidgetForPhoneGap/issues"`

Reply:
Done